config/locales/idv/en.yml

```diff
@@ -173,6 +173,8 @@ en:
         JavaScript to continue this process.'
     gpo:
       alert_info: 'We sent a letter with your one-time code to:'
+      alert_spam_warning_html: 'We are unable to send more letters. The most recent
+        letter was sent on <strong>%{date_letter_was_sent}</strong>.'
```

Comment:
Is this a permanent ban, or do we want to add "until %{end_date}" ?

Reply:
> Is this a permanent ban, or do we want to add "until %{end_date}" ?

@Kamal-Munshi  ... pls advise on @soniaconnolly inquiry ... thanks 🙏🏿

Reply:
@soniaconnolly - i reached out to @Kamal-Munshi and @tahineemay  and t[hey are good to move forward as is](https://github.com/18F/identity-idp/pull/9303#issuecomment-1749368772
)